### ISSUE TITLE
Error while executing query with a graphQL Int parameter type

### ISSUE THREAD

**Issue by cmenetrier** [graphql-java-generator/graphql-maven-plugin-project#82]
com.graphql_java_generator.util.GraphqlUtils#getClass is not handling Standard GraphQL type `Int` properly and therefore try to find a custom type

**Comment by etienne-sf** [comment]
Hum,

Can you provide the schéma and the request ?

Étienne

**Comment by etienne-sf** [comment]
Or perhaps you are using a bind variable or a GraphQL variable ?

**Comment by etienne-sf** [comment]
Ok, there is a workaround:

Use a Bind Variable, instead of the int value being written in the query.


For instance, with this query: `"{withOneMandatoryParamDefaultValue (intParam: ?param)  {}}"`, you can use the `param` bind variable this way:

```java
YourReturnType response = request.execQuery("param", 1).getWithOneMandatoryParamDefaultValue();
```

Etienne

**Comment by etienne-sf** [comment]
This is corrected in the 1.16 release

Can you check that ?

Etienne

**Comment by cmenetrier** [comment]
All good, thanks
